components/actionbar/index.css

```diff
@@ -9,51 +9,18 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
+@import "./definitions.css";
+
 .spectrum-ActionBar {
 	--spectrum-actionbar-height: var(--spectrum-action-bar-height);
```

Comment:
@castastrophe IMO, at this point, it feels like it makes more sense to define _all_ custom properties in the `definitions.css` file rather than just the ones that refer to "global" (or "private") Spectrum Tokens. I think it makes it a little easier to explain _what_ goes into `definitions.css` (every custom property definition!) rather than explaining the reasoning behind how one would pick and choose. I'm happy to be convinced otherwise, of course! What do you think?

Reply:
Yes, I think that makes a lot of sense!